### ISSUE TITLE
Bring in fipio

### ISSUE THREAD

**Issue by mikejohnson51** [mikejohnson51/AOI#19]
I would like to ping @program-- to work with me on replacing USAboudnaries data with `fipio`.

**Comment by program--** [comment]
Hey @mikejohnson51! I'm finishing up 1.1.0 for fipio right now (going to forgo `spress` for now, at the expense of ZCTA geometry resolution) and am planning to push it up to CRAN today or tomorrow. I'll open a PR as soon as I merge fipio 1.1.0 in its repo -- sound good?

**Comment by mikejohnson51** [comment]
sounds great! Hey, so how would you do the following in fipio?

```r
AOI = AOI::aoi_get(state = "OH", county = 'all')
```

This type of request is the one possible limitation I have seen to this integration...

**Comment by program--** [comment]
See here: https://github.com/program--/fipio/blob/eba5eba635c144c88ad4c0bbbf3f16b2feb9c089/R/fipio.R#L12-L47
This feature isn't up on CRAN yet but will be

I'm going to add `"all"` and `"conus"` options just for AOI :smile:

**Comment by program--** [comment]
@mikejohnson51 I submitted `fipio` 1.1.0 yesterday, but I had to resubmit again due to some URLs in the README that needed to be fixed to their redirects... I missed the window by a day as the CRAN submission team is returning this message on submission:
```
The submission team is on vacation till Jan 3, 2022.
During this time the submission of packages is not possible.
Sorry for any inconvenience.
```
:disappointed: I'll still work on the AOI PR but this is going to push it back till Jan 3rd.

**Comment by mikejohnson51** [comment]
Not a problem! Thanks @program-- :)

**Comment by mikejohnson51** [comment]
I believe this is closed? @program--

**Comment by program--** [comment]
@mikejohnson51 I think so 😄

**Comment by mikejohnson51** [comment]
me too.  Closed with 894af3a6ee862985bcaa4e07e05285c74971525f
